actioncable/lib/action_cable/subscription_adapter/test.rb

```diff
@@ -0,0 +1,24 @@
+module ActionCable
+  module SubscriptionAdapter
+    # == Test adapter for Action Cable
+    class Test < Base
+      def broadcast(channel, payload)
+        transmissions(channel) << payload
+      end
+
+      def transmissions(channel)
+        channels_data[channel] ||= []
+      end
+
+      def clear_messages(channel)
+        channels_data[channel] = []
+      end
+
+      private
+
+      def channels_data
+        @channels_data ||= {}
+      end
```

Comment:
Can you remove the space under private and then indent private methods like you did in the other file?